### PR TITLE
message%: add support for text colors

### PR DESCRIPTION
**Pull request by Bogdanp**
The CI failure is expected since the GTK change depends on a change to `draw-lib`.

### CODE REVIEW

File: gui-doc/scribblings/gui/message-class.scrbl [hunk 1]
```diff
@@ -65,5 +74,20 @@ The same as @xmethod[window<%> set-label] when @racket[label] is a
 Otherwise, sets the bitmap label for a bitmap message.
  @bitmaplabeluseisbm[label] @|bitmapiforiglabel|
 
-}}
+}
+
+@defmethod[(set-label-color [color color%]) void?]{
```
Comment (sorawee) [inline gui-doc/scribblings/gui/message-class.scrbl:79]: This should be `(is-a?/c color%)`, right?

File: gui-doc/scribblings/gui/message-class.scrbl [hunk 2]
```diff
@@ -10,12 +10,13 @@ A message control is a static line of text or a static bitmap. The
...
 
-@defconstructor[([label (or/c label-string? (is-a?/c bitmap%) 
+@defconstructor[([label (or/c label-string? (is-a?/c bitmap%)
                               (or/c 'app 'caution 'stop))]
-                 [parent (or/c (is-a?/c frame%) (is-a?/c dialog%) 
+                 [parent (or/c (is-a?/c frame%) (is-a?/c dialog%)
                                (is-a?/c panel%) (is-a?/c pane%))]
                  [style (listof (or/c 'deleted)) null]
                  [font (is-a?/c font%) normal-control-font]
+                 [color (is-a?/c color%) _unset]
```
Comment (sorawee) [inline gui-doc/scribblings/gui/message-class.scrbl:19]: Would it make sense to make this `(or/c #f (is-a?/c color%))` and default to `#f`?

File: gui-doc/scribblings/gui/message-class.scrbl (hunk 1 above)
Comment (Bogdanp) [reply gui-doc/scribblings/gui/message-class.scrbl:79]: Whoops, yes, thanks!

**Comment by mflatt** [comment]
This looks great! Two things I noticed when I tried it:

* The initialization argument and color-setting methods should probably accept a string and convert it with `(send the-color-database find-color ....)`.

* The initialization argument is `color`, but the methods are `...label-color`. It would be better to make those the same, and I'd be inclined to go with `...color` instead of making the initialization argument `label-color`.

**Comment by Bogdanp** [comment]
Thanks! I've made those changes.

**Comment by mflatt** [comment]
Thanks! Ok if I squash while merging?

**Comment by Bogdanp** [comment]
Yup!

**Pull request merged**